src/IndexData.ts

```diff
@@ -34,6 +34,11 @@ export const indexDataSchema = z.object({
     latest: z.coerce
         .string()
         .describe("The latest version of the mod to be pushed to the subscribers"),
+    dependencies:
+        z.array(z.coerce.string().regex(
+            /^[a-z0-9-]+$/,
```

Comment:
[Note] Not sure how we can validate that this ID is within our registry. We may not want to validate that as it could be a private registry item dependent on something public.

Reply:
Can we maybe just add validation to prevent enable if dependency is not subscribed?

Reply:
I think the system fails silently